deps/v8/third_party/inspector_protocol/CodeGenerator.py

```diff
@@ -59,13 +59,13 @@ def init_defaults(config_tuple, path, defaults):
         jinja_dir = arg_options.jinja_dir
         if not jinja_dir:
             raise Exception("jinja directory must be specified")
-        output_base = arg_options.output_base
+        output_base = arg_options.output_base.decode('utf-8')
         if not output_base:
             raise Exception("Base output directory must be specified")
         config_file = arg_options.config
         if not config_file:
             raise Exception("Config file name must be specified")
-        config_base = os.path.dirname(config_file)
+        config_base = os.path.dirname(config_file).decode('utf-8')
```

Comment:
No changes should be made to the deps. Those changes should only happen upstream.

Reply:
Agreed.

Reply:
Changes to this effect were already made upstream-upstream, btw: https://chromium-review.googlesource.com/c/deps/inspector_protocol/+/574858 … ping @ak239